packages/dashboard/lib/DashboardServer.ts

```diff
@@ -117,6 +117,7 @@ export class DashboardServer {
         analyticsSet: true,
         analyticsMessageDateTime: Date.now()
       });
+      _res.send();
```

Comment:
Two things here:
1. This variable was prepended with `_` because it was unused, so using it here means we should remove `_`.
2. I don't think we need to `send`? Does it work without? In any case this can probably be `res.end`

Reply:
Woo. Yes let me try those.

Reply:
Does `res.end` send a response to the client?

Reply:
Yes, client should get a 200.